modules/op-adding-triggers.adoc

```diff
@@ -0,0 +1,151 @@
+----
++
+Alternatively, you can create `TriggerBinding` directly from the `pipelines-tutorial` Git repository:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/01_binding.yaml
+----
+
+. Copy the content of the following sample `TriggerTemplate` YAML file and save it:
++
+----
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: vote-app
+spec:
+  params:
+  - name: git-repo-url
+    description: The git repository url
+  - name: git-revision
+    description: The git revision
+    default: master
+  - name: git-repo-name
+    description: The name of the deployment to be created / patched
+
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: $(params.git-repo-name)-git-repo-$(uid)
+    spec:
+      type: git
+      params:
+      - name: revision
+        value: $(params.git-revision)
+      - name: url
+        value: $(params.git-repo-url)
+
+  - apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: $(params.git-repo-name)-image-$(uid)
+    spec:
+      type: image
+      params:
+      - name: url
+        value: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/$(params.git-repo-name):latest
+
+  - apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      name: build-deploy-$(params.git-repo-name)-$(uid)
+    spec:
+      serviceAccountName: pipeline
+      pipelineRef:
+        name: build-and-deploy
+      resources:
+      - name: git-repo
+        resourceRef:
+          name: $(params.git-repo-name)-git-repo-$(uid)
+      - name: image
+        resourceRef:
+          name: $(params.git-repo-name)-image-$(uid)
+      params:
+      - name: deployment-name
+        value: $(params.git-repo-name)
+----
+
+. Create the `TriggerTemplate`:
++
+----
+$ oc create -f <triggertemplate-yaml-file-name.yaml>
+----
++
+Alternatively, you can create `TriggerTemplate` directly from the `pipelines-tutorial` Git repository:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/02_template.yaml
+----
+
+. Copy the contents of the following sample `EventListener` YAML file and save it:
++
+----
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: vote-app
+spec:
+  serviceAccountName: pipeline
+  triggers:
+  - bindings:
+    - name: vote-app
+    template:
+      name: vote-app
+----
+
+. Create `EventListener`:
++
+----
+$ oc create -f <eventlistener-yaml-file-name.yaml>
+----
++
+Alternatively, you can create `EvenListener` directly from the `pipelines-tutorial` Git repository:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_event_listener.yaml
+----
++
+[NOTE]
+====
+EventListener setups a Service. However, this Service needs to be exposed as an {product-title} route to make it publicly accessible.
```

Comment:
the `EventListener` sets up

Reply:
This note seems to be the why for the next step. Suggest you add it to the next sentence as part of the step, not sure it needs to be added as a note.